cdbsearch.py

```diff
@@ -370,6 +370,11 @@ def search(self, board, depth):
         if depth > 15:
             self.reprobe_PV(board, minicache[bestmove])
 
+        if bestscore > 25000:
+            bestscore -= 1
+        elif bestscore < -25000:
+            bestscore += 1
```

Comment:
Could the same logic not also be applied to TBwins? Then could change 25000 to 20000.

Reply:
I guess changing this to 15000 would mean that the script will try to find shorter paths to TBwins, whereas as it is any distance to a TBwin is treated equally (as in current main/master). Hence I think this would be a good change, and become the main thrust of this PR.